lib/Common/Memory/MarkContext.inl

```diff
@@ -46,6 +46,11 @@ bool MarkContext::AddTrackedObject(FinalizableObject * obj)
 #endif
 
 template <bool parallel, bool interior, bool doSpecialMark>
+#if defined(__has_feature)
+#if __has_feature(address_sanitizer)
```

Comment:
You may introduce a NO_SANITIZE definition into CommonPal instead? Same for other declarations below